lib/formatLoaderError.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+/**
+ * Returns a formatted loader error either as a string or separated by an "and" or commas.
+ * @param {array|string} data
+ * @returns {array|string}
+ */
+
+const formatLoaderError = data => {
+	if (!data.length) return [];
+	if (typeof data === "string") {
+		return `${data}`;
+	}
+	if (Array.isArray(data)) {
+		const len = data.length;
+		if (len === 1) {
+			return `${data[0] || ""}`;
+		}
+		if (len === 2) {
+			return data.join(" and ");
+		}
+		return `${data
+			.map((d, i) => {
+				return i !== data.length - 1 ? `${d}` : `and ${d}`;
+			})
+			.join(", ")}`;
```

Comment:
Won't this make the string to have both `, ` and `and` in front of the last element?

Reply:
You're right. Let me update.

Reply:
Removing this and following Tobias above suggestion.